src/Monitors/Models.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Collection\Monitors;
+
+use Collection\Monitors\Monitors;
+use PDO;
+
+class Models
+{
+    private PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+    
+    
+    public function getAllProducts(): array
+    {
+        $query = $this->db->prepare("SELECT * FROM `monitors`;");
+
+        $query->execute();
+
+        $data = $query->fetchAll();
+
+        $products = []; // We define an empty array to contain the result
+
+        foreach ($data as $productData) {
```

Comment:
productData should probably be monitorData

Reply:
changed to monitorData